mlflow/sparkml.py

```diff
@@ -0,0 +1,116 @@
+"""
+MLflow integration for SparkML models.
+
+SparkML models are saved and loaded using native SparkML persistence.
+The models can be exported as pyfunc for out-of Spark deployment or it can be loaded back as Spark
+Transformer in order to score it in Spark. The pyfunc flavor instantiates SparkContext internally
+and reads the input data as Spark DataFrame prior to scoring.
+
+Note: SparkML model can not be exported as spark_udf as those two formats are incompatible.
```

Comment:
This is out of date now, right? Did you try it as a python UDF? I think it will work.

Reply:
Yes I did run it. The spark context is not a problem but lack of column names inside of spark_udf is. Since most spark ml models require column names it still does not work. I could add column names to the model or just this flavor and set them on the Dataframe before it is passed to the model. There is no way to extract feature names from PipelineModel, so user would have to provide those explicitly.